hack/ci/e2e.sh

```diff
@@ -46,13 +46,17 @@ install_kind() {
     TMP_DIR=$(mktemp -d)
     # ensure bin dir
     mkdir -p "${TMP_DIR}/bin"
-    # kind requries go modules now
-    export GO111MODULE="on"
+    # kind requires go modules now
     # if we have a kind checkout, install that to the tmpdir, otherwise go get it
+    # TODO: this script should probably go away anyhow, but this isn't exactly
+    # module mode ...
     if [[ $(go list sigs.k8s.io/kind) = "sigs.k8s.io/kind" ]]; then
-        env "GOBIN=${TMP_DIR}/bin" go install sigs.k8s.io/kind
+        DIR="${PWD}"
+        cd ./../../sigs.k8s.io/kind
+        make install INSTALL_PATH="${TMP_DIR}/bin"
+        cd "${DIR}"
```

Comment:
I'm sure this can be done more cleanly, but it appears to work fine, and it's CI specific anyhow.

Reply:
Well yes, and the working for for this script is Kubernetes, to clarify.

Reply:
updated, I fixed the makefile's path stuff and then had this use `"$(dirname "${BASH_SOURCE[0]}")"` to locate itself and compute the path.

Reply:
should be much cleaner now